challenge_serving_drinks/src/challenge_serving_drinks/sd_states.py

```diff
@@ -207,7 +203,7 @@ def __init__(self, robot, unavailable_drink_designator, max_tries=3, max_queries
         """ Initialization method
 
         :param robot: robot api object
-        :param unavailable_drink_designator: designator in which the unavailable drink (entity type) is stored
+        :param unavailable_drink_designator: (VariableDesignator) in which the unavailable drink is stored
```

Comment:
The entity type, ID or just the object? What kind type should the designator accept?